game/src/plugins/skills/mining/src/mining.plugin.kts

```diff
@@ -198,8 +198,8 @@ class ProspectingAction(val m: Player, val p: Position, val ore: Ore) : Distance
 on {ObjectActionMessage::class}
         .where {option == 1}
         .then {
-            if (ORES.contains(id)) {
-                it.startAction(MiningAction(it, id, this.position, ORES.get(id)!!))
+            if (lookupOreRock(id) != null) {
+                it.startAction(MiningAction(it, id, this.position, lookupOreRock(id)!!))
             } else {
                 System.out.println("Unknown ore: " + id)
```

Comment:
Can we remove this debug message? If the ore doesn't exist then it means the player clicked a different object, we don't want to spam the console every time that happens